repair_type_sequence/tests/test_repair_type.py

```diff
@@ -7,7 +7,7 @@
 
 class TestRepairType(TransactionCase):
     def setUp(self, *args, **kwargs):
-        super(TestRepairType, self).setUp(*args, **kwargs)
```

Comment:
Please use `setUpClass`

Reply:
There is only one test, so it will not change anything because both functions will only be executed once.